app/views/feedback/admin.html.erb

```diff
@@ -1,5 +1,7 @@
 <div class="cf-app-msg-screen cf-app-segment cf-app-segment--alt">
-    <h2>Current Feedback</h2>
+	<div class="cf-push-left">
+    <h2>Current Feedback <span class="cf-lead-paragraph"><span class ="count"><%= count %></span> New feedback forms today</span></h2>
```

Comment:
Try
```
</span>New feedback submissions today:  </span><%= @count %>
```

also, doesn't need to be inside the `h2` block

Reply:
This looks like fine UI:
<img width="945" alt="screen shot 2017-10-24 at 4 11 50 pm" src="https://user-images.githubusercontent.com/18075411/31965992-29436a00-b8d7-11e7-92c9-2f82e8e0cc48.png">


you can add some margin/padding to move it away from the table, but it's your call, you are the main user